lib/hammer_cli/apipie/command.rb

```diff
@@ -47,11 +47,10 @@ def help
     protected
 
     def send_request
-      if resource && resource.has_action?(action)
-        resource.call(action, request_params, request_headers, request_options)
-      else
-        raise HammerCLI::OperationNotSupportedError, "The server does not support such operation."
+      unless resource && resource.has_action?(action)
+        raise HammerCLI::OperationNotSupportedError, _('The server does not support such operation.')
       end
+      extended_data(resource.call(action, *extended_request))
```

Comment:
I decided to do it here instead of small fix for `hammer-cli-foreman` to get it work there (since `hammer-cli-foreman` overrides `print_data` in https://github.com/theforeman/hammer-cli-foreman/blob/16de4c49ceb6d40d1d13dc0ffc200f8119f02389/lib/hammer_cli_foreman/commands.rb#L369).

Reply:
I see, makes sense.